src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -779,6 +779,7 @@ override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitWhileUntilLoopSta
 override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitWithStatement(Microsoft.CodeAnalysis.Semantics.IWithStatement operation) -> void
 override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitYieldBreakStatement(Microsoft.CodeAnalysis.Semantics.IReturnStatement operation) -> void
 override Microsoft.CodeAnalysis.SyntaxNode.ToString() -> string
+static Microsoft.CodeAnalysis.CaseInsensitiveComparison.StartsWith(string value, string possibleStart) -> bool
```

Comment:
Yes, I don't think there is any issue. It's just that Neal asked to be pinged as FYI for any public API change.